examples/multi_objective/botorch_simple.py

```diff
@@ -0,0 +1,54 @@
+from botorch.settings import suppress_botorch_warnings
+from botorch.settings import validate_input_scaling
+from botorch.test_functions.multi_objective import C2DTLZ2
+import torch
+
+import optuna
+
+
+# C2-DTLZ2 minimization objective function with a single constraint.
+# It is negated to be a maximization problem since BoTorch otherwise assumes maximization.
+_OBJECTIVE = C2DTLZ2(dim=3, num_objectives=2, negate=True)
```

Comment:
Ah, it's because you haven't change the `directions` when creating the study. I changed the example to automatically adjust the length of the argument.

Reply:
I confirmed that the example can be executed with various pairs of `dim` and `n_objectives`. Let me resolve this conversation.